pl_bolts/datamodules/vocdetection_datamodule.py

```diff
@@ -152,16 +152,20 @@ def prepare_data(self) -> None:
         VOCDetection(self.data_dir, year=self.year, image_set="val", download=True)
 
     def train_dataloader(
-        self, batch_size: int = 1, image_transforms: Union[List[Callable], Callable] = None
+        self,
+        batch_size: int = 1,
+        transforms: List[Callable] = [],
```

Comment:
It's not considered good practice to have an empty list as default.

Reply:
That's true. For some reason I wanted to minimize changes outside the YOLO class, but you're right. I'll change the default value to None.